### PR TITLE
Add more postgres support

### PR DESCRIPTION
**Pull request by drew-gross**
Also cleans up authData handling a bit. Note that in order to create the username/email unique constraint in Postgres, that means the _User class needs to exist, so we are now creating it when the server starts.

### CODE REVIEW

File: src/Controllers/SchemaController.js [hunk 1]
```diff
@@ -241,21 +240,22 @@ const convertAdapterSchemaToParseSchema = ({...schema}) => {
...
     delete schema.fields._hashed_password;
     schema.fields.password = { type: 'String' };
   }
 
   return schema;
 }
 
-const injectDefaultSchema = schema => ({
-  className: schema.className,
+const injectDefaultSchema = ({className, fields, classLevelPermissions}) => ({
```
Comment (flovilmart) [inline src/Controllers/SchemaController.js]: nice

**Comment by peterdotjs** [comment]
nice! looks super clean.

**Comment by lifeisfunny** [comment]
Is this a Postgres database adapter ? Can I use to replace mongodb ? When is the official release ?

**Comment by lifeisfunny** [comment]
Can I try it ? Super excited !

**Pull request merged**